ERROR - 2023-12-15 00:19:05 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`e-libraryxpert`.`fines`, CONSTRAINT `fines_loan_id_foreign` FOREIGN KEY (`loan_id`) REFERENCES `loans` (`id`) ON DELETE SET NULL ON UPDATE CASCADE) in C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fi...', 0)
#1 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fi...')
#2 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fi...')
#3 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1800): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fi...', NULL, false)
#4 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2191): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\laragon\www\CI4_E-LibraryAradh\app\Database\Seeds\FineSeeder.php(32): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\Seeder.php(146): App\Database\Seeds\FineSeeder->run()
#7 C:\laragon\www\CI4_E-LibraryAradh\app\Database\Seeds\LoanSeeder.php(56): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\Seeder.php(146): App\Database\Seeds\LoanSeeder->run()
#9 C:\laragon\www\CI4_E-LibraryAradh\app\Database\Seeds\Seeder.php(18): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#10 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\Seeder.php(146): App\Database\Seeds\Seeder->run()
#11 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Commands\Database\Seed.php(77): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#12 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run(Array)
#13 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\CLI\Console.php(46): CodeIgniter\CLI\Commands->run('db:seed', Array)
#14 C:\laragon\www\CI4_E-LibraryAradh\spark(102): CodeIgniter\CLI\Console->run()
#15 {main}
ERROR - 2023-12-15 00:21:48 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`e-libraryxpert`.`fines`, CONSTRAINT `fines_loan_id_foreign` FOREIGN KEY (`loan_id`) REFERENCES `loans` (`id`) ON DELETE SET NULL ON UPDATE CASCADE) in C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fi...', 0)
#1 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fi...')
#2 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fi...')
#3 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1800): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fi...', NULL, false)
#4 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2191): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\laragon\www\CI4_E-LibraryAradh\app\Database\Seeds\FineSeeder.php(32): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\Seeder.php(146): App\Database\Seeds\FineSeeder->run()
#7 C:\laragon\www\CI4_E-LibraryAradh\app\Database\Seeds\LoanSeeder.php(56): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\Seeder.php(146): App\Database\Seeds\LoanSeeder->run()
#9 C:\laragon\www\CI4_E-LibraryAradh\app\Database\Seeds\Seeder.php(18): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#10 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\Seeder.php(146): App\Database\Seeds\Seeder->run()
#11 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Commands\Database\Seed.php(77): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#12 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run(Array)
#13 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\CLI\Console.php(46): CodeIgniter\CLI\Commands->run('db:seed', Array)
#14 C:\laragon\www\CI4_E-LibraryAradh\spark(102): CodeIgniter\CLI\Console->run()
#15 {main}
ERROR - 2023-12-15 00:49:37 --> mysqli_sql_exception: Column count doesn't match value count at row 1 in C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `lo...', 0)
#1 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `lo...')
#2 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `lo...')
#3 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1800): CodeIgniter\Database\BaseConnection->query('INSERT INTO `lo...', NULL, false)
#4 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2191): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\laragon\www\CI4_E-LibraryAradh\app\Database\Seeds\LoanSeeder.php(64): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\Seeder.php(146): App\Database\Seeds\LoanSeeder->run()
#7 C:\laragon\www\CI4_E-LibraryAradh\app\Database\Seeds\Seeder.php(18): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\Seeder.php(146): App\Database\Seeds\Seeder->run()
#9 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Commands\Database\Seed.php(77): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#10 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run(Array)
#11 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\CLI\Console.php(46): CodeIgniter\CLI\Commands->run('db:seed', Array)
#12 C:\laragon\www\CI4_E-LibraryAradh\spark(102): CodeIgniter\CLI\Console->run()
#13 {main}
ERROR - 2023-12-15 11:04:22 --> mysqli_sql_exception: Duplicate entry '356a192b7913b04c54574d18c28d46e6395428ab' for key 'uid' in C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `lo...', 0)
#1 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `lo...')
#2 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `lo...')
#3 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1800): CodeIgniter\Database\BaseConnection->query('INSERT INTO `lo...', NULL, false)
#4 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2191): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\laragon\www\CI4_E-LibraryAradh\app\Database\Seeds\LoanSeeder.php(74): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\Seeder.php(146): App\Database\Seeds\LoanSeeder->run()
#7 C:\laragon\www\CI4_E-LibraryAradh\app\Database\Seeds\Seeder.php(18): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\Seeder.php(146): App\Database\Seeds\Seeder->run()
#9 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Commands\Database\Seed.php(77): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#10 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run(Array)
#11 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\CLI\Console.php(46): CodeIgniter\CLI\Commands->run('db:seed', Array)
#12 C:\laragon\www\CI4_E-LibraryAradh\spark(102): CodeIgniter\CLI\Console->run()
#13 {main}
ERROR - 2023-12-15 11:08:50 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`e-libraryxpert`.`fines`, CONSTRAINT `fines_loan_id_foreign` FOREIGN KEY (`loan_id`) REFERENCES `loans` (`id`) ON DELETE SET NULL ON UPDATE CASCADE) in C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fi...', 0)
#1 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fi...')
#2 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fi...')
#3 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1800): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fi...', NULL, false)
#4 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2191): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\laragon\www\CI4_E-LibraryAradh\app\Database\Seeds\FineSeeder.php(32): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\Seeder.php(146): App\Database\Seeds\FineSeeder->run()
#7 C:\laragon\www\CI4_E-LibraryAradh\app\Database\Seeds\LoanSeeder.php(48): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\Seeder.php(146): App\Database\Seeds\LoanSeeder->run()
#9 C:\laragon\www\CI4_E-LibraryAradh\app\Database\Seeds\Seeder.php(18): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#10 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\Seeder.php(146): App\Database\Seeds\Seeder->run()
#11 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Commands\Database\Seed.php(77): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#12 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run(Array)
#13 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\CLI\Console.php(46): CodeIgniter\CLI\Commands->run('db:seed', Array)
#14 C:\laragon\www\CI4_E-LibraryAradh\spark(102): CodeIgniter\CLI\Console->run()
#15 {main}
ERROR - 2023-12-15 11:14:00 --> mysqli_sql_exception: Duplicate entry '356a192b7913b04c54574d18c28d46e6395428ab' for key 'uid' in C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `lo...', 0)
#1 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `lo...')
#2 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `lo...')
#3 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1800): CodeIgniter\Database\BaseConnection->query('INSERT INTO `lo...', NULL, false)
#4 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2191): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\laragon\www\CI4_E-LibraryAradh\app\Database\Seeds\LoanSeeder.php(74): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\Seeder.php(146): App\Database\Seeds\LoanSeeder->run()
#7 C:\laragon\www\CI4_E-LibraryAradh\app\Database\Seeds\Seeder.php(18): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\Seeder.php(146): App\Database\Seeds\Seeder->run()
#9 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Commands\Database\Seed.php(77): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#10 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run(Array)
#11 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\CLI\Console.php(46): CodeIgniter\CLI\Commands->run('db:seed', Array)
#12 C:\laragon\www\CI4_E-LibraryAradh\spark(102): CodeIgniter\CLI\Console->run()
#13 {main}
ERROR - 2023-12-15 11:25:11 --> mysqli_sql_exception: Duplicate entry '77de68daecd823babbb58edb1c8e14d7106e83bb' for key 'uid' in C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `lo...', 0)
#1 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `lo...')
#2 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `lo...')
#3 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1800): CodeIgniter\Database\BaseConnection->query('INSERT INTO `lo...', NULL, false)
#4 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2191): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\laragon\www\CI4_E-LibraryAradh\app\Database\Seeds\LoanSeeder.php(54): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\Seeder.php(146): App\Database\Seeds\LoanSeeder->run()
#7 C:\laragon\www\CI4_E-LibraryAradh\app\Database\Seeds\Seeder.php(18): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\Seeder.php(146): App\Database\Seeds\Seeder->run()
#9 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Commands\Database\Seed.php(77): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#10 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run(Array)
#11 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\CLI\Console.php(46): CodeIgniter\CLI\Commands->run('db:seed', Array)
#12 C:\laragon\www\CI4_E-LibraryAradh\spark(102): CodeIgniter\CLI\Console->run()
#13 {main}
ERROR - 2023-12-15 11:26:38 --> mysqli_sql_exception: Duplicate entry '77de68daecd823babbb58edb1c8e14d7106e83bb' for key 'uid' in C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `lo...', 0)
#1 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `lo...')
#2 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `lo...')
#3 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1800): CodeIgniter\Database\BaseConnection->query('INSERT INTO `lo...', NULL, false)
#4 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2191): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\laragon\www\CI4_E-LibraryAradh\app\Database\Seeds\LoanSeeder.php(54): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\Seeder.php(146): App\Database\Seeds\LoanSeeder->run()
#7 C:\laragon\www\CI4_E-LibraryAradh\app\Database\Seeds\Seeder.php(18): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Database\Seeder.php(146): App\Database\Seeds\Seeder->run()
#9 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\Commands\Database\Seed.php(77): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#10 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run(Array)
#11 C:\laragon\www\CI4_E-LibraryAradh\vendor\codeigniter4\framework\system\CLI\Console.php(46): CodeIgniter\CLI\Commands->run('db:seed', Array)
#12 C:\laragon\www\CI4_E-LibraryAradh\spark(102): CodeIgniter\CLI\Console->run()
#13 {main}
CRITICAL - 2023-12-15 12:13:00 --> Undefined array key "identities"
in APPPATH\Views\users\index.php on line 44.
 1 APPPATH\Views\users\index.php(44): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "identities"', 'C:\\laragon\\www\\CI4_E-LibraryAradh\\app\\Views\\users\\index.php', 44)
 2 SYSTEMPATH\View\View.php(228): include('C:\\laragon\\www\\CI4_E-LibraryAradh\\app\\Views\\users\\index.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('users/index', [], true)
 5 APPPATH\Controllers\Users\UsersController.php(38): view('users/index', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Users\UsersController->index()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users\UsersController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\CI4_E-LibraryAradh\\public\\index.php')
